rules/strict-equality.js

```diff
@@ -0,0 +1,66 @@
+const eqeqeq = require("eslint/lib/rules/eqeqeq");
+const astUtils = require("eslint/lib/rules/utils/ast-utils");
+
+// - Prefer double equals when comparing with undefined
+// - Prefer undefined over null
+// - Prefer triple equals everywhere else
+module.exports = {
+  meta: {
+    type: "suggestion",
+    fixable: "code",
```

Comment:
Also not fixable

Reply:
Actually it was when I wrote it, but I decided to remove the fixing because it's probably best to rely on manual judgement.

That said, the `eqeqeq` plugin that we fall through to can return a fix in certain situations I believe.